README.md

```diff
@@ -83,6 +94,22 @@ You can easily access this file via the Claude Desktop app by navigating to `Cla
   }
 }
 ```
+For example, it may looks something this like:
+
+```json
+{
+  "mcpServers": {
+    "perses-mcp": {
+      "command": "/Users/user-bob/mcp-server/bin/mcp-server",
+      "args": ["--perses-server-url", "https://demo.perses.dev"],
+      "env": {
+        "PERSES_TOKEN": "1234567890123456789012345678901234567890"
+      }
+    }
+  }
+}
+```
+
```

Comment:
Is this a duplicate of https://github.com/zhuje/mcp-server/blob/5c87b80da0ac362b2bf3d66401c8d12bb43fba2c/README.md?plain=1#L81-L95